apps/parse/lib/parse/stop_times.ex

```diff
@@ -35,6 +35,12 @@ defmodule Parse.StopTimes do
       arrival_time: convert_time(row["arrival_time"], row["drop_off_type"]),
       departure_time: convert_time(row["departure_time"], row["pickup_type"]),
       stop_sequence: String.to_integer(row["stop_sequence"]),
+      stop_headsign:
+        if row["stop_headsign"] == "" do
+          nil
+        else
+          copy(row["stop_headsign"])
+        end,
```

Comment:
I think there's an `optional_copy` helper?

Reply:
Looks like there's one defined locally in `vehicle_positions.ex`, but I can easily move that into the `Parse.Helpers` module.